pkg/option/config.go

```diff
@@ -965,6 +973,11 @@ func (c *DaemonConfig) IsFlannelMasterDeviceSet() bool {
 	return len(c.FlannelMasterDevice) != 0
 }
 
+// IsSubsysMetricEnabled returns true if the given subsystem metric is enabled.
+func (c *DaemonConfig) IsSubsysMetricEnabled(subsystem uint64) bool {
```

Comment:
What's the benefit of a uint64 mask over a struct with booleans? It seems simpler to be able to:
```
if option.Config.EnabledMetrics.BpfSyscalls {
```